.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: d
+addons:
+  apt:
+    sources:
+    # older versions of Make don't support the wildcard rules and will create folders instead of files
+    - make
+script: make
+before_deploy:
+- echo -e "Host digitalmars.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
+- openssl aes-256-cbc -K $encrypted_1daeb42065ec_key -iv $encrypted_1daeb42065ec_iv
+  -in deploy_rsa.enc -out deploy_rsa -d
```

Comment:
These two lines need to be replaced with the ones that result from the `travis encrypt` command.